CONTRIBUTING.md

```diff
@@ -92,10 +103,23 @@ Look for issues without the label `assigned` on it. This indicates that someone
 To start, comment on the issue asking if you can work on it. A maintainer will tag your username to the issue and apply the `assigned` label on it.
 
 ## Setting up a Development Environment
+
 For hassle-free code contribution, follow the steps below. NOTE: I mostly work in a Linux environment using a basic text editor (vim). Your set-up could be different. If it is, please feel free to submit an issue regarding your development environment and a pull request to add to this documentation.
 
+## Visual Studio Code Development Container
```

Comment:
This header should be the same (i.e. `###`) as `Before you clone the project`, `Setting up a development environment on Mac and Windows` and others below it, no? Otherwise, it looks like the others are sub headers related to the VS Code setup, which they are not. 

I also might request that you add more clear steps of what folks need to do to get up and going (assuming folks are completely new to VS Code). I am happy to write this part up, too, FYI but I think we need to be more explicit in how to get started. I'd like to see steps like...

```
1. Once you have installed Visual Studios Code, Docker and The VS Code Remote - Contains extension, open VS Code and click the "Remote Explorer" Icon on the left. 
2. Click the plus sign next to `CONTAINERS` and select `Open Folder in container`. Select *your forks* cloned repository from the drop down  menu.
3. ... the rest of the steps assuming I could get it to work :P
```

Reply:
I've updated the instructions. I _think_ directing users to the icon in the bottom left is quicker than going via the "Remote Explorer" menu down the left side, but I'm not sure. I would appreciate your input here.

I've also skipped the cloning instructions here. I think that it's better to either give it its own section, or direct users to GitHub documentation.